src/redux/selectors/locationsSelectors.js

```diff
@@ -0,0 +1,12 @@
+import { createSelectorCreator, defaultMemoize } from 'reselect';
+import { areLocationsTheSame } from '../../common/helpers';
+
+const locationsSelector = state => state.locations.locations;
+export const isLocationsLoadingSelector = state => state.locations.isLoading;
+
+// I realize this is a bit of overkill, just wanted to try out the selectors
+const createDeepEqualSelector = createSelectorCreator(defaultMemoize, areLocationsTheSame);
```

Comment:
so you can't apply your selector for other means within locations feature anymore. You are creating a super specific selector fabric that can only work based on areLocationsTheSame comparator. I don't think this is a good idea.

Reply:
agree